app/commands/put_content_with_links.rb

```diff
@@ -48,6 +48,8 @@ def create_or_update_live_content_item!(draft_content_item)
       )
 
       LiveContentItem.create_or_replace(attributes) do |item|
+        clear_space_for(item)
```

Comment:
Seems to me that we only need to do this when item itself is new, so maybe we can do `clear_space_for(item) if item.new_record?`